focus-ios/focus-ios-tests/SmokeTest.xctestplan

```diff
@@ -34,6 +34,8 @@
         "SettingAppearanceTest\/testOpenInSafari()",
         "TrackingProtectionMenu\/testInactiveProtectionSidebar()",
         "TrackingProtectionSettings",
+        "URLValidationTest",
```

Comment:
I would say this is an important test that should run as part of the Smoketest

Reply:
Updated